lib/v8debugapi.js

```diff
@@ -198,7 +199,7 @@ module.exports.create = function(logger_, config_, fileStats_) {
             });
           });
 
-      listeners[num] = listener;
+      listeners[num] = { enabled: true, listener: listener };
```

Comment:
nit: This is not actionable right now, but since I've seen this object signature created in several times throughout the agent might it be a good idea to turn it into a shared class?

e.g.
```JS
function ListenerContainer ( enabled=false, listener=null ) {
  this.enabled_ = enabled;
  this.listener_ = listener;
}

ListenerContainer.setIsEnabled ( enabled=false ) {
  this.enabled_ = enabled;

  return this;
}

ListenerContainer.setListener = function ( listener=null ) {
  this.listener_ = listener;

  return this;
}
```